src/components/WYSIWYGInput/index.tsx

```diff
@@ -295,6 +300,9 @@ export const WYSIWYGInput = forwardRef<WYSIWYGInputRef, WYSIWYGInputProps>(
             {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events */}
             <div
               className={classnames('pui-wysiwyg-editor', {
+                'h-44': size === 'small',
+                'h-56': size === 'medium',
+                'h-96': size === 'large',
```

Comment:
Instead of adding multiple `if` here, prefer using a "map object", similar to this:
https://github.com/perimetre/ui/blob/master/src/components/Button/index.tsx#L10

It would end up looking like
```ts
const sizeClassnameMap = {  small: 'h-44',  medium: 'h-56',  large: 'h-96' };
```

For three reasons:
- We already have that patten
- It removes the need of having three ifs, to having only a single reference
- This is not scalable, imagine if we need to have 10 sizes, we'd have 10 ifs.